src/app/customer/GuestForm.tsx

```diff
@@ -32,6 +33,7 @@ const GuestForm: FunctionComponent<GuestFormProps & WithLanguageProps & FormikPr
     canSubscribe,
     checkoutButtons,
     isContinuingAsGuest,
+    isLoading,
```

Comment:
I wonder if we should just collapse these 2 into `isLoading`. The form doesn't really care why it is loading, just that it is... 😕 what do you think?

cc: @davidchin

Reply:
sounds good to me, but I would like to hear @davidchin 's opinion.